edb/schema/functions.py

```diff
@@ -633,6 +633,9 @@ class Function(CallableObject, s_abc.Function):
     initial_value = so.SchemaField(
         expr.Expression, default=None, compcoef=0.4, coerce=True)
 
+    session_only = so.SchemaField(
```

Comment:
You also want to add a validation that all overloaded functions have the same `session_only` setting.  Similarly you will later do the same for `volatility`.

Reply:
Yes, good catch. Check added.